schemas/v1.0/azure.yaml.json

```diff
@@ -135,9 +140,125 @@
                                 "description": ""
                             }
                         }
+                    },
+                    "scripts": {
+                        "type": "object",
+                        "title": "Definition of command hooks for developer integration",
+                        "descriptions": "Command hooks should match `azd` command names prefixed with `pre` or `post` depending on when the script should execute relative to the command",
+                        "minProperties": 1,
+                        "additionalProperties": {
```

Comment:
I assume not all property names will work--only those that correspond to actual AZD commands (`postdeploy` yes, but `prefixallbugs` not yet ☺️ ). Should the schema give the user a hint what "script kinds" are available?

@savannahostrowski @bwateratmsft Should we consider a VS Code extension gesture to "add Azure Dev CLI script"? It could let you specify the kind and PowerShell/Bash, then make appropriate entry in azure.yaml, create corresponding file (e.g. under `${workspaceRoot}/scripts` and open it in the IDE for editing. That would be a separate PR I assume.

Reply:
Correct, only matching commands will work.  Does JSON schema support these types of hints? Do you have any docs I could refer to.